modules/oxide/src/less/theme/components/revisionhistory/revisionhistory.less

```diff
@@ -56,26 +56,31 @@
   }
 
   .tox-revisionhistory__sidebar {
-    border-left: @revisionhistory-border;
+    display: flex;
+    flex-direction: column;
     height: 100%;
-    max-width: 360px;
+    min-width: @revisionhistory-sidebar-minwidth;
+    width: 436px;
 
     .tox-revisionhistory__sidebar-title {
-      border-bottom: @revisionhistory-border;
+      background-color: @background-color;
+      box-shadow: 0 4px 8px 0 rgba(34, 47, 62, .1);
       color: @revisionhistory-text-color;
-      font-size: @revisionhistory-sidebar-title-font-size;
+      font-size: 20px;
```

Comment:
you add variables for border styles but remove one for font size? 🤔

Reply:
Didn't expect the original value is already 20px. Reverted it!